### ISSUE TITLE
Add Accessibility Testing Plan to Documentation Site

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#6944]
## Description 
As a member of the mobile app team, I want to ensure that I can access the [VA.gov Mobile Accessibility Testing Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/testing/Accessibility-Test-Plan.md) on the documentation site.

## Acceptance Criteria
  

-  [x] Add the [VA.gov Mobile Accessibility Testing Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/testing/Accessibility-Test-Plan.md) to the documentation site.
-  [x] Update the mobile accessibility testing plan link on the [Accessibility](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/UX/Foundations/Accessibility/) page of the documentation site.

**Comment by bischoffa** [comment]
@brea11y wanted to see if this work is planned for the next sprint (Its in the next sprint column) if so can you update update the sprint tag?

**Comment by brea11y** [comment]
@bischoffa – This should be completed this sprint. The PR has two approvals. Just waiting on the merge and I'll get this one closed out. I updated the sprint tag to this current sprint.
